prometheus/osfamilymap.yaml

```diff
@@ -31,66 +33,108 @@ Debian:
         name: prometheus
       alertmanager:
         name: prometheus-alertmanager
+        service:
+          name: prometheus-alertmanager
```

Comment:
Please don't add `name` otherwise it breaks

Reply:
Sorry, but I fail to see what 'breaks' if I add the `service:name` attribute. Mind you elaborate?

I added the defaults in the Debian family to the values that packages use so no matter what what method you use to install (`archives` or `packages`), the service file has a consistent name and location.

I fixed the [default/archive test pillar](https://github.com/saltstack-formulas/prometheus-formula/pull/44/files#diff-fb6eae9eb240e4b08c915a9da45f52f4R96) to be consistent with the [archive method test, common to all the distros](https://github.com/saltstack-formulas/prometheus-formula/blob/master/test/integration/default/controls/service_spec.rb#L6) that we're using.

Isn't it OK? Am I missing something?

Reply:
> Isn't it OK? Am I missing something?

yes.

This is Debian in archive mode : https://travis-ci.com/github/saltstack-formulas/prometheus-formula/jobs/377094793

We can see here that it download node exporter : https://travis-ci.com/github/saltstack-formulas/prometheus-formula/jobs/377094793#L3546

Tests fail : 

```sh
×  prometheus services: should be running (3 failed)
   ×  Service node_exporter is expected to be enabled
   expected that `Service node_exporter` is enabled
   ×  Service node_exporter is expected to be running
   expected that `Service node_exporter` is running
   ×  Port 9100 is expected to be listening
   expected `Port 9100.listening?` to return true, got false
```
why?

because : 

it doesn't know `prometheus-node-exporter` : https://travis-ci.com/github/saltstack-formulas/prometheus-formula/jobs/377094793#L2767

Reply:
I had to remove them to make the tests pass : https://github.com/saltstack-formulas/prometheus-formula/commit/b03a1ccf04c124354f2485167a8e1e905172d780

Reply:
And now we fallback on [35](https://github.com/saltstack-formulas/prometheus-formula/issues/35)

Reply:
>  so no matter what what method you use to install (`archives` or `packages`), the service file has a consistent name and location.

this is the issue. it **does matter**.

Reply:
The issue was caused because of the way [service file name](https://github.com/saltstack-formulas/prometheus-formula/blob/d65858a15426fb91d8e8d862f81c25620e658f4a/prometheus/archive/install.sls#L89) is managed in the `archive/install.sls` file.

I think now I fixed that, and also added a bunch of tests to verify different `service` name are used, if provided.

Reply:
> The issue was caused because of the way [service file name](https://github.com/saltstack-formulas/prometheus-formula/blob/d65858a15426fb91d8e8d862f81c25620e658f4a/prometheus/archive/install.sls#L89) is managed in the `archive/install.sls` file.

I see that